### PR TITLE
📝 add translation (zh-tw) and code block for vimscript

### PR DESCRIPTION
**Pull request by Hsins**
I refer to this article for sculpting my own `vimrc` and translate it to my native language meanwhile. Moreover, I add the logo and badges to beautify the `README.md`

### CODE REVIEW

**Comment by romainl** [comment]
Thank you.

I have noticed a few things:

- I would prefer the following sentence to be returned to its original form:

  > Guidelines for Sculpting Your Very Own Vim/Neovim Configuration (`.vimrc`)

  for a few reasons:

  - While some or all of the information found in this document may also be applicable to Neovim, I can and will only guarantee its correctness with regard to Vim. This means that there should be no mention of Neovim anywhere.
  - I am not a big fan of titlecase so I would prefer the sentence to use normal casing.
  - Also, this document goes a bit beyond the `vimrc` file so "(`.vimrc`)" is misleading.

- "Read in Other Avaiable Translations" should be "Read in other available translations".
- I don't think it is worth it to hide that short section behind a widget.
- Emojis have no place in formal documents so both the "roadblock" and the two flags should be removed. Furthermore, using flags to denote language is a very common error.
- The Vim syntax highlighting on the code blocks of the English version doesn't work because the whole block, including the fences is indented by 4 spaces. That indenting must be removed.

**Comment by JoyceBabu** [comment]
GitHub supports linking to images within the repository in README file. Might be better to commit the logo too to this repo.

**Comment by Hsins** [comment]
@romainl 

I modified the points you mentioned and accept the advice from @JoyceBabu to add the hyperlink of logo images in `README.me`.

**Comment by JoyceBabu** [comment]
@Hsins I think you have misunderstood my suggestion. I mean that you can commit the `svg` file too to the repository. Then you can link to the commited logo by using a relative path.

For example, in the following page the image `controller_brief.svg` [[1]] is also part of the repository.

https://github.com/potherca-blog/StackOverflow/tree/master/question.13808020.include-an-svg-hosted-on-github-in-markdown#linking-to-relative-files-works-but-obviously-only-httpsgithubcom

For example, if you commit the file as `logo.svg`, then you can embed the image in the Markdown with

```
![Alt text](./logo.svg)
```
or
```
<img src="./logo.svg">
```

By including the logo in the repository, you can avoid dependency on an external service (img.shields.io in this case)

[1]: https://github.com/potherca-blog/StackOverflow/blob/master/question.13808020.include-an-svg-hosted-on-github-in-markdown/controllers_brief.svg

**Comment by Hsins** [comment]
@JoyceBabu

I'm sure know that we can embed the image in the Markdown file by using the relative path. It's considered bad practice to upload the binaries file (eg. `*.png`, `*.jpg`... in this case) to GitHub repos of pure documents (I think). And that's why I stored the logo image on `imgur.com`.

Another way to avoid the dependecy on an external service is to upload the logo image to GitHub in such this issue thread:

image

(so that we can use the url `https://user-images.githubusercontent.com/26391143/115075688-0e2bde00-9f2e-11eb-8876-b669c41fbd68.png` in `README.md`)

Moreover, I don't think it's necessary to upload the `*.svg` files of badges. After all, the service from `img.shields.io` is stable and used everywhere on GItHub.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
+  </a>
+</p>
+
+<!-- Title and Description -->
+<div align="center">
+
 # IDIOMATIC VIMRC
 
-Guidelines for sculpting your very own `~/.vimrc`.
+🔨 _Guidelines for sculpting your very own `~/.vimrc`_
```
Comment (romainl) [inline README.md]: Please revert this line to its original state:

    Guidelines for sculpting your very own `~/.vimrc`.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,35 @@
...
+
+</div>
+
+<div align="center">
```
Comment (romainl) [inline README.md]: I feel that the HTML is unnecessary. The text below should be enough:

    This document is available in the following languages:

    - [English](./README.md)
    - [Traditional Chinese](./README-zh_TW.md)

File: README.md (hunk 1 above)
Comment (Hsins) [reply README.md]: reverted in the amending commit

**Pull request closed without merging**